src/draw.typ

```diff
@@ -306,6 +306,30 @@
   ),)
 }
 
+/// Draw body on z-index layer
+///
+/// This can be used to draw elements behind already drawn elements by
+/// using a lower layer value (i.e -1).
+/// 
+/// - layer (integer, float): Z-Index to draw the children at
+/// - body (draw, function): Child elements or function (`ctx => draw-cmds`)
```

Comment:
Do we have a dedicated type description to mean "elements", if not could you add one and use it in `canvas` and where ever else it needs.

Reply:
We use elements at other places, I changed it to that.